llvm/test/Transforms/FunctionSpecialization/function-specialization-track-ptr-return.ll

```diff
@@ -0,0 +1,147 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --include-generated-funcs --version 5
```

Comment:
Can you remove the "function-specialization-" prefix from the file name? It is already under a folder named accordingly.

Reply:
Ah yep good point, Done.